examples/python/easy_start_demo/README.md

```diff
@@ -0,0 +1,28 @@
+## Demo for using gRPC in Python
```

Comment:
Being in a directory named `easy_start_demo`, I expect that this will be the first example that people click on when taking a first look at the repo. These people may have no point of reference for what gRPC is or how it works. That being the case, we either
 1. Expand this `README` or
 2. Change the directory name

I would also suggest splitting it into two files: `README.en.md` and `README.cn.md`

Reply:
I will change the directory name,  and rebuild the README.